### ISSUE TITLE
How to see net product price?

### ISSUE THREAD

**Issue by vladosmolovskiy** [allegro/allegro-api#8881]
Hello, I'm wondering is it possible to see net price in order? Because in web version it is visible, but in offer request all prices comes with VAT included

**Comment by PrzemyslawLukanowski** [comment]
No, it is not possible, but recently we have added information about [the tax rate](https://developer.allegro.pl/news/orders-we-return-the-tax-rate-set-in-the-offer-jn8Yg6Em3FZ) included in the offer.
